server/data/quotes.json

```diff
@@ -339,6 +339,10 @@
     "author": "Johann Wolfgang von Goethe",
     "sentence": "A man who is ignorant of foreign languages is also ignorant of his own language."
   },
+  {
+    "author": "Johann Wolfgang von Goethe",
+    "sentence": "He who knows no foreign languages knows nothing of his own."
```

Comment:
This quote has the same meaning as the quote above it.

_A man who is ignorant of foreign languages is also ignorant of his own language._

Hope you can contribute a different quote.